src/workload.rs

```diff
@@ -487,6 +497,18 @@ impl WorkloadInformation {
     }
 }
 
+impl Default for Box<dyn CertificateProvider> {
```

Comment:
We shouldn't have fake test things in prod code, makes it possible to accidentally use this

Reply:
done